client/src/about/containers/AboutContainer.tsx

```diff
@@ -1,15 +1,10 @@
 import { connect } from 'react-redux';
```

Comment:
Can we just delete this whole file since it's not doing anything anymore? Then just remember to import about instead of aboutcontainer in the routingcontainer